WordPress/Classes/Networking/RemotePublicizeService.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+@objc public class RemotePublicizeService : NSObject
+{
+    public var connectURL = ""
+    public var detail = ""
+    public var icon = ""
+    public var jetpackSupport = false
+    public var jetpackModuleRequired = "publicize"
```

Comment:
I think `var` is best since there are publicize-like services we might explore using at some point.  Tho for now its value it will most likely always be `publicize` after assignment.  Maybe it just looks weird to have a non-empty string assigned as a default?